tests/test_worker_db.py

```diff
@@ -0,0 +1,64 @@
+from frontera.core.models import Request, Response
+from frontera.worker.db import DBWorker
+from frontera.settings import Settings
+from frontera.core.components import States
+
```

Comment:
There is a ``available_partitions`` and ``mark_ready``/``mark_busy`` mechanisms in message bus https://github.com/scrapinghub/frontera/blob/master/frontera/core/messagebus.py#L137. They are used mainly in DBW, and it would be nice to test them.